larq/quantizers_test.py

```diff
@@ -48,47 +48,99 @@ def test_invalid_usage():
         lq.quantizers.get("unknown")
 
 
-@pytest.mark.parametrize("name", ["ste_sign", "approx_sign", "swish_sign"])
-def test_binarization(name):
+def generate_real_values_with_zeros(low=-2, high=2, shape=(4, 10)):
+    real_values = np.random.uniform(low, high, shape)
+    real_values = np.insert(real_values, 1, 0, axis=1)
+    return real_values
+
+
+@pytest.mark.parametrize(
+    "fn",
+    [
+        "ste_sign",
+        lq.quantizers.SteSign(),
+        "approx_sign",
+        "swish_sign",
+        lq.quantizers.SwishSign(),
+    ],
+)
+def test_xnor_binarization(fn):
     x = tf.keras.backend.placeholder(ndim=2)
-    f = tf.keras.backend.function([x], [lq.quantizers.get(name)(x)])
+    f = tf.keras.backend.function([x], [lq.quantizers.get(fn)(x)])
     binarized_values = np.random.choice([-1, 1], size=(2, 5))
     result = f([binarized_values])[0]
     np.testing.assert_allclose(result, binarized_values)
 
-    real_values = np.random.uniform(-2, 2, (2, 5))
+    real_values = generate_real_values_with_zeros()
     result = f([real_values])[0]
     assert not np.any(result == 0)
-    assert np.all(result[result < 0] == -1)
-    assert np.all(result[result >= 0] == 1)
+    assert np.all(result[real_values < 0] == -1)
+    assert np.all(result[real_values >= 0] == 1)
```

Comment:
Pro tip: `assert True` will always make your tests pass 🙈 

Thanks for catching this 💯